src/world/partner/bombette.c

```diff
@@ -239,81 +242,83 @@ ApiStatus func_802BD758_3184A8(Evt* evt, s32 isInitialCall) {
             npc->yaw = atan2(npc->pos.x, npc->pos.z, playerStatus->position.x, playerStatus->position.z);
             suggest_player_anim_clearUnkFlag(ANIM_Mario_10002);
             evt->functionTemp[0] = 21;
-        case 21:
-            if (playerStatus->actionState == 0x15 || playerStatus->actionState == 0x16) {
-                disable_npc_blur(npc);
-                evt->functionTemp[0] = 7;
-                break;
-            }
 
-            if ((playerStatus->flags & 2)) {
+        case 21: {
```

Comment:
Is this manual scope at the beginning of case 21 necessary? It seems strange and shouldn't be